week1-3/playlist.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="playlist.css">
+    <title>Time playlist</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Questrial&display=swap" rel="stylesheet">
+    <link href='//spoqa.github.io/spoqa-han-sans/css/SpoqaHanSansNeo.css' rel='stylesheet' type='text/css'>
+</head>
+<body>
+    
+    <header id="pageTop">&#127911; Music Playlist &#127911;</header>
+    <section class="preview-wrapper">
+        <img src="/asset/preview_1.jpg" alt="for_lovers_who_hesitate">
+        <img src="/asset/preview_2.jpg" alt="flower">
+        <img src="/asset/preview_3.jpg" alt="time_of_our_life">
+        <img src="/asset/preview_4.jpg" alt="fish">>
+        <img src="/asset/preview_5.jpg" alt="youngblood">
+        <img src="/asset/preview_6.jpg" alt="four_seasons">
+        <img src="/asset/preview_7.jpg" alt="lose_you_to_love_me">
+    </section>
+    <nav>
+        <section class="nav-wrapper">
+            <section class="nav-explanation">
+                <div class="helper-text">계절을 클릭하여 이동할 수 있어요</div>
+            </section>
+            <section class="nav-items">
+            
+                <a href="#1" class="nav-items-link">봄/여름</a>
+                <a href="#2" class="nav-items-link">가을</a>
+                <a href="#3" class="nav-items-link">겨울</a>
+            </section>
+        </section>
+    </nav>
```

Comment:
```suggestion
    <nav>
        <div class="nav-wrapper">
            <p class="helper-text">계절을 클릭하여 이동할 수 있어요</p>
            <ul class="nav-items">
                <li><a href="#1" class="nav-items-link">봄/여름</a></li>
                <li><a href="#2" class="nav-items-link">가을</a></li>
                <li><a href="#3" class="nav-items-link">겨울</a></li>
            </ul>
        </div>
    </nav>
```

구조에 대해서는 승희언니가 너무 자세하게 써줬으니! 결론적으로 이런 구조가 되면 좀 더 적합하겠죠?! 
그리고 혹시 굳이 디자인적으로 필요가 없다면 nav-wrapper도 없어도 됩니당!